docs/install_guides/installing-openvino-conda.md

```diff
@@ -1,10 +1,118 @@
-# Install OpenVINO™ Runtime from Anaconda Cloud
+# Install OpenVINO™ Runtime from Anaconda Cloud {#openvino_docs_install_guides_installing_openvino_conda}
 
-* [Install OpenVINO Runtime from an Archive File](installing-openvino-from-archive-linux.md)
-* [Install OpenVINO from PyPI](installing-openvino-pip.md)
-* [Install OpenVINO with Docker](installing-openvino-docker-linux.md)
-* [Build From Source](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md)
+@sphinxdirective
 
-The other installation methods are temporarily unavailable.
+.. note:
+
+   Installing OpenVINO Runtime from Anaconda Cloud is recommended for C++ developers, as it provides only the C++ Runtime API.
+
+   If you work with Python, the PyPI package includes everything you need to develop and deploy OpenVINO. 
+   Visit the :doc:`Install OpenVINO from PyPI <openvino_docs_install_guides_installing_openvino_pip>` page for instructions 
+   on how to install OpenVINO Runtime for Python using PyPI.
+
+.. tab:: System Requirements
+
+  | Full requirement listing is available in:
+  | `System Requirements Page <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html>`__
+
+.. tab:: Processor Notes
+
+  | Not all Intel CPUs include integrated graphics processors.  
+  | See `Product Specifications <https://ark.intel.com/>`__ for information about your hardware.
+
+.. tab:: Software
+
+   There are many ways to work with Conda. Before you proceed, learn more about it on the
+   `Anaconda distribution page <https://www.anaconda.com/products/individual/>`__
+
+
+
+
+Installing OpenVINO Runtime with Anaconda Package Manager
+############################################################
+
+1.  Set up the Anaconda environment (Python 3.7 used as an example):
+   
+   .. code-block:: sh
+
+      conda create --name py37 python=3.7
+
+   .. code-block:: sh
+
+      conda activate py37
+
+2. Update it to the latest version:
+   
+   .. code-block:: sh
+
+      conda update --all
+
+.. don't we need this one?:
+
+   3. Install pre-requisites:
+   
+      .. code-block:: sh
+   
+         conda install numpy
```

Comment:
Is this not handled via package dependencies automatically?

Reply:
I hope it is :) Just want to make sure. The fewer steps we have, the better.